unbound/USAGE-client.md

```diff
@@ -0,0 +1,58 @@
+
+# Usage of the client side container
+
+The post quantum key exchange [openssl](https://github.com/open-quantum-safe/openssl) variant and [getdns](https://getdnsapi.net/) were installed to query the DNS server with a post quantum key exchange algorithm.
+## Installation
+Assuming you have docker [installed](https://docs.docker.com/install) on your machine all command below will launch dns server docker.
+
+Run getdns container:
+```bash
+    docker run --network unbound-test --interactive --tty --hostname getdns --name getdns -it openquantumsafe/getdns
+```
+After running all the command above a container will open with getdns running with openssl post quantum variant.
+
+## Usage
+
+In the file [Dockerfile](getdns-docker/Dockerfile#L45) the environment variable  TLS_DEFAULT_GROUPS is set "p384_kyber768:X25519" to force for p384_kyber768 key exchange. Other key exchange algorithms can be used, find more algorithm in the [list of available post quantum key exchange algorithms](https://github.com/open-quantum-safe/boringssl#key-exchange). 
```

Comment:
This is the USAGE file, i.e., the file intended for deployment at docker hub: Readers there expect the image to run and not be build (again). Thus please change this sentence to omit any reference to the Dockerfile and use the ´-e´ parameter to `docker run` instead to set the env var.

Reply:
I change it and give an example in both file on how to change the key exchange algorithm.